inc/saiport.h

```diff
@@ -4113,7 +4113,1313 @@ typedef enum _sai_port_serdes_attr_t
     SAI_PORT_SERDES_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
     /** End of custom range base */
-    SAI_PORT_SERDES_ATTR_CUSTOM_RANGE_END
+    SAI_PORT_SERDES_ATTR_CUSTOM_RANGE_END,
+
+    /** Public custom range base value */
+    SAI_PORT_SERDES_ATTR_PUBLIC_CUSTOM_RANGE_START = 0x20000000,
```

Comment:
this range is reserved for DASH

Reply:
Changed to `0x30000000` to avoid conflict with DASH.

And will discuss more in the meeting.